torch/onnx/_internal/exporter/_onnx_program.py

```diff
@@ -151,16 +187,17 @@ def __call__(self, *args, **kwargs) -> Sequence[torch.Tensor]:
 
         # We don't expect non-tensor as inputs
         ort_input = {
-            k.name: v.numpy(force=True)
+            k.name: _to_ort_value(v)
```

Comment:
What happens if the user only have CPU?

Reply:
That's ok. It should still work as shown by our CI tests in pytorch